src/components/interfaces/MOBILE_API.xml

```diff
@@ -6100,6 +6100,9 @@
 
     <function name="OnRCStatus" functionID="OnRCStatusID" messagetype="notification">
       <description>Issued by SDL to notify the application about remote control status change on SDL</description>
+      <param name="allowed" type="Boolean" mandatory="false">
+        <description>If "true" - RC is allowed; if "false" - RC is disallowed.</description>
```

Comment:
@LitvinenkoIra I would also specify in description cases when this parameter is absent in the notification. 
Please clarify it with @yang1070 and if he is OK with it. Add it to requirements and to notification description.